scripts/in_container/_in_container_utils.sh

```diff
@@ -561,6 +561,27 @@ function check_missing_providers() {
     rm "$LIST_OF_DIRS_FILE"
 }
 
+function rename_packages_if_needed() {
```

Comment:
I thought this want needed and we could get setuptools to output the right name directly with a flag to the `egginfo` command?

Reply:
Unfortunately not :(. Setuptools/egg_info keep the name of packge and version in sync and we need the case where they are different.

We need to rename the packages to contain *rc*  when we want to do the trick where package name contains `rc` but internally it has no tag (so where we publish RCs to SVN) . In this case `--tag-build` is not helpful (we do not use it to keep the "target" version" in the package metadata). 

The `--tag-build` command is only used in case of PyPI RC* packages (where the name + version have to match) as well as when we release (potentially) alpha/beta (in this case also name of package and metadata version are in sync). But the RCs we put in SVN have no `--tag-build` at all but have to be named wih rc.

We could potentially simply change our convention and keep the "target" package name in sync with version in rc* directories, but I think that would be confusing when you download it - so better to keep it as is IMHO.

No egg_info command is helpful here.

```
Options for 'egg_info' command:
  --egg-base (-e)   directory containing .egg-info directories (default: top
                    of the source tree)
  --tag-date (-d)   Add date stamp (e.g. 20050528) to version number
  --tag-build (-b)  Specify explicit tag to add to version number
  --no-date (-D)    Don't include date stamp [default]
```